apps/docs/assets/scripts/theme-handler.js

```diff
@@ -0,0 +1,40 @@
+function getTheme() {
+    return localStorage.getItem('theme') || 'auto';
+}
+
+function isDark() {
+    if (theme === 'auto') {
+        return window.matchMedia('(prefers-color-scheme: dark)').matches;
+    }
+    return theme === 'dark';
+}
+
+function setTheme(newTheme) {
+    theme = newTheme;
+
+    localStorage.setItem('theme', theme);
+    button.icon = isDark() ? 'dark-mode-line' : 'light-mode-line';
```

Comment:
```suggestion
    button.icon = isDark() ? 'dark-mode-solid' : 'light-mode-line';
```
I think the solid one looks better in the dark theme